hist/hist/src/TFormula.cxx

```diff
@@ -3209,7 +3172,10 @@ void TFormula::GradientPar(const Double_t *x, Double_t *result)
       //    }
       // }
       args[1] = &result;
-      (*fGradFuncPtr)(0, 2, args, /*ret*/nullptr); // We do not use ret in a return-void func.
+      (*FuncPtr)(0,
+                 2,
+                 args, /*ret*/
+                 nullptr); // We do not use ret in a return-void func.
```

Comment:
Maybe undo the clang-format changes and keep the formatting as close as possible to what was it in the master.